GameModes/Pathfinder/Pathfinder.csproj

```diff
@@ -1,4 +1,26 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <ItemGroup>
+    <Compile Update="**\*.axaml.cs">
+      <DependentUpon>%(Filename)</DependentUpon>
+    </Compile>
+    <AvaloniaResource Include="**\*.axaml">
+      <SubType>Designer</SubType>
+    </AvaloniaResource>
   </ItemGroup>
-</Project>
+  <ItemGroup>
+    <AvaloniaResource Remove="Views\PathfinderCharacterView.axaml" />
```

Comment:
Is this intentionally removing the view we just added?

Reply:
When you do the "dotnet new avalonia.usercontrol" this is what it does.  If you take the remove away, you get really weird errors.

Reply:
I just removed the original add.  I'm not sure why it adds all of them and then deletes them.  But it seems to work if you don't have _any_ AvaloniaResources?

Reply:
*shrugs* Oh well, I'm super confused, but it all seems to work, even after a git clean without this... so... gone?